shuup/default_reports/reports/orders.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2019, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.default_reports.forms import OrderLineReportForm, OrderReportForm
+from shuup.default_reports.mixins import OrderLineReportMixin, OrderReportMixin
+from shuup.reports.report import ShuupReportBase
+from shuup.utils.i18n import get_locally_formatted_datetime
+
+
+class OrdersReport(OrderReportMixin, ShuupReportBase):
+    identifier = "orders_report"
+    title = _("Orders Report")
+    form_class = OrderReportForm
+
+    filename_template = "orders-report-%(time)s"
+    schema = [
+        {"key": "order_num", "title": _("Order ref.")},
+        {"key": "order_date", "title": _("Order date")},
+        {"key": "customer", "title": _("Customer")},
+        {"key": "status", "title": _("Status")},
+        {"key": "order_line_quantity", "title": _("Order line quantity")},
+        {"key": "payment_status", "title": _("Payment status")},
+        {"key": "shipment_status", "title": _("Shipment status")},
+        {"key": "order_total_amount", "title": _("Total")},
+    ]
+
+    def get_data(self):
+        data = []
+        orders = self.get_objects(paid=False)
+
+        for order in orders:
+            data.append({
+                "order_num": order.identifier,
+                "order_date": get_locally_formatted_datetime(order.order_date),
+                "status": order.status,
+                "order_line_quantity": order.lines.count(),
```

Comment:
maybe this could consider only products?

Reply:
Hm, right now a user can see products only if he selects product type in OrderLineReport.

Reply:
But this will show the total number of lines no matter what